packages/neo-one-ts-utils/src/class_.ts

```diff
@@ -314,7 +315,10 @@ function getExtendorsWorker(
       }
 
       const clause = node_.getParent(parent) as ts.Node | undefined;
-      if (clause === undefined || !ts.isHeritageClause(clause) || !heritage.isExtends(clause)) {
+
+      if (clause && ts.isExpressionWithTypeArguments(clause)) {
+        // do nothing and continue.
+      } else if (clause === undefined || !ts.isHeritageClause(clause) || !heritage.isExtends(clause)) {
```

Comment:
This is a strange pattern. I'd prefer you combined it into one `if` statement like so:
```
if (clause === undefined || !ts.isHeritageClause(clause) || !heritage.isExtends(clause) && !(clause === undefined && ts.isExpressionWithTypeArguments(clause))) {
   return acc;
}
```

Or whatever makes sense with that logic